### PR TITLE
fix(tests): add totp functional tests

### PR DESCRIPTION
**Pull request by vbudhram**
From https://github.com/mozilla/fxa-content-server/pull/5958#issue-173346455, the adds some functional tests for TOTP.

### CODE REVIEW

**Comment by vbudhram** [comment]
@mozilla/fxa-devs r?

File: tests/functional.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = [
   'tests/functional/sign_in.js',
   'tests/functional/sign_in_cached.js',
   'tests/functional/sign_in_blocked.js',
+  'tests/functional/sign_in_totp.js',
```
Comment (vladikoff) [inline tests/functional.js]: @vbudhram should we add this to circle ci tests as well?

Comment (vbudhram) [reply tests/functional.js]: Added!

**Comment by vbudhram** [comment]
@vladikoff Added to circle and moved functional oauth TOTP tests into oauth file. Think this should be good!

File: circle.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ dependencies:
     - npm i --production
     - CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build
     - nohup bash -c "CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt serverproc:dist &"
-    - npm install intern xmlhttprequest firefox-profile request sync-exec convict yargs leadfoot
+    - npm install intern xmlhttprequest firefox-profile request sync-exec convict yargs leadfoot otplib
```
Comment (vladikoff) [inline circle.yml]: @vbudhram 
image

could you add the otplib to `run.sh` and `install-npm-deps.sh`.

**Pull request merged**